packages/static/kairos-overlay-files/files/system/oem/12_nvidia.yaml

```diff
@@ -11,9 +11,9 @@ stages:
         permissions: 0644
         owner: 0
         group: 0
-      # TODO: This is no longer read. This should be moved to cloud-init file.
```

Comment:
This is still true, no?

Reply:
for the time being but it's what we need to fix in this story